src/harness/compilerRunner.ts

```diff
@@ -183,8 +183,9 @@ class CompilerBaselineRunner extends RunnerBase {
                                 jsCode += result.declFilesCode[i].code;
                             }
                         }
-
+                        options.optimizationEntrypoint = undefined;
                         let declFileCompilationResult = harnessCompiler.compileDeclarationFiles(toBeCompiled, otherFiles, result, function (settings) {
+                            delete tcSettings["optimizationEntrypoint"];

```

Comment:
Ah! I tried removing this line, ran tests, and was reminded why this needed to be done. When the declaration files get checked, the entrypoint ts file isn't part of the compilation, resulting in an extraneous error in the output.